c7n/resources/rds.py

```diff
@@ -509,8 +509,7 @@ class Stop(BaseAction):
 
     schema = type_schema('stop')
 
-    # permissions are unclear, and not currrently documented or in iam gen
-    permissions = ("rds:RebootDBInstance",)
+    permissions = ("rds:StopDBInstance")
```

Comment:
this needs a trailing comma to turn into an tuple

Reply:
Thanks, just updated this.  Sorry, a Python noob here...

Reply:
all good, thanks